### PR TITLE
GithubTeacher adds Manchester, CT

### LINKED ISSUE #45: @Vijaybabu27 follows Github flow
# Manchester, CT
- [x] create a **branch**
- [x] add some **commits**
- [ ] open a pull request
- [ ] collaborate
- [ ] merge ` with 👍

### PR DESCRIPTION
**Pull request by vijaybabu27**
This Pull request is to add Manchester, CT to the list of Hometowns

@githubteacher what kind of things would you like to do with your son in Manchester, CT

This PR closed #45

### CODE REVIEW

File: Vijaybabu27-Manchester.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+### Restaurants
+
+- Godavari
+- Dharani
+- Nautia
+
+### Attractions / Places to see
+
+- Wicham Park
+- Dianousar State Park
```
Comment (manners15) [inline Vijaybabu27-Manchester.md]: @vijaybabu27   I love visiting Parks!

Comment (vijaybabu27) [inline Vijaybabu27-Manchester.md]: @manners15  good to know

Comment (vijaybabu27) [inline Vijaybabu27-Manchester.md]: @manners15  added few more parks list

Comment (vijaybabu27) [inline Vijaybabu27-Manchester.md]: @manners15  i added one more park please approve my changes

**Pull request merged**